### PR TITLE
Fix 0x00 postgres

### PR DESCRIPTION
**Pull request by lunny**
When migrating https://gitea.com/xorm/xorm into a gitea instance with postgres, it will report a postgres text column cannot include `0x00` char error.

This PR will replace all issue title/content and comment content `0x00` as empty before migrating.

### CODE REVIEW

File: models/issues/issue.go [hunk 1]
```diff
@@ -901,8 +902,17 @@ func InsertIssues(ctx context.Context, issues ...*Issue) error {
 	return committer.Commit()
 }
 
+func replaceNullByte(s string) string {
```
Comment (harryzcy) [inline models/issues/issue.go:905]: Maybe `removeNullBytes`?

**Review by harryzcy — COMMENTED** [review]
Why is there null characters in the first place? Is frontend sending back null characters?

**Comment by wxiaoguang** [comment]
It is a quite hacky patch, I do not think it is right.

For example, why don't you patch the "update" functions? There are a lot of user-provided contents which may contain "\x00"

**Comment by lunny** [comment]
Close as it's not ideal. Open an issue #28890 and close this one.

**Pull request closed without merging**